File: PHP/KodeProgram/Aksesoris.php
<?php
// Aksesoris.php
require_once 'Elektronik.php';

class Aksesoris extends Elektronik {
    private $jenis;
    private $warna;
    private $compatibility;
    private $foto_produk; // Atribut khusus PHP

    public function __construct($id, $nama, $harga, $brand, $garansi, $stok, $jenis, $warna, $compatibility, $foto_produk) {
        parent::__construct($id, $nama, $harga, $brand, $garansi, $stok);
        $this->jenis = $jenis;
        $this->warna = $warna;
        $this->compatibility = $compatibility;
        $this->foto_produk = $foto_produk;
    }

    public function getKategori() {
        return "Aksesoris";
    }

    public function getDetail() {
        return "Jenis:{$this->jenis}, Warna:{$this->warna}, {$this->compatibility}";
    }
    
    public function getFotoProduk() {
        return $this->foto_produk;
    }
}

File: Java/KodeProgram/Aksesoris.java
// Aksesoris.java
// Berisi definisi class Aksesoris, turunan dari Elektronik.
class Aksesoris extends Elektronik {
    private String jenis;
    private String warna;
    private String compatibility;

    public Aksesoris(int id, String nama, double harga,
                     String brand, int garansi, int stok,
                     String jenis, String warna, String compatibility) {
        super(id, nama, harga, brand, garansi, stok);
        this.jenis = jenis;
        this.warna = warna;
        this.compatibility = compatibility;
    }

    @Override
    public String getKategori() { return "Aksesoris"; }

    @Override
    public String getDetail() {
        return "Jenis:" + jenis + ", Warna:" + warna + ", " + compatibility;
    }
}